Cargo.toml

```diff
@@ -8,7 +8,8 @@ repository = "https://github.com/gevulotnetwork/gvltctl"
 description = "Gevulot Control CLI"
 
 [dependencies]
-gevulot-rs = "0.1.3-pre.3"
+# gevulot-rs = "0.1.3-pre.3"
+gevulot-rs = { git = "https://github.com/gevulotnetwork/gevulot-rs.git", rev = "e4094641c487da48af5bd1b49c2bac50a48b21e7" }
```

Comment:
Btw I think it would be nice to use `[patch]` here to emphasize that this is only a temporary change, which will be reverted in the future (because it is essentially a patch to published version):

```toml
[patch.crates-io]
gevulot-rs = { git = "https://github.com/gevulotnetwork/gevulot-rs.git", rev = "e4094641c487da48af5bd1b49c2bac50a48b21e7" }
```
```

Reply:
Another this is that: when using `[patch]` you will patch all used `gevulot-rs`-s including the one used from `mia-installer`. So you will avoid compiling `gevulot-rs` twice and use a single patched version instead.

Reply:
This is cool :)